providers/standard/tests/unit/standard/sensors/test_time.py

```diff
@@ -58,6 +58,35 @@ def test_target_time_aware_dag_timezone(self):
             op = TimeSensor(task_id="test", target_time=aware_time)
             assert op.target_datetime.tzinfo == timezone.utc
 
+    @pytest.mark.parametrize(
+        "current_datetime, server_timezone",
+        [
+            ("2025-01-26 23:00:00", "UTC"),
+            ("2025-01-27 07:00:00", DEFAULT_TIMEZONE),
```

Comment:
This test would break if someone changes `DEFAULT_TIMEZONE` without noticing it depends on this case relies on exactly 7+ hours of offset. It would be much better to create a timezone explicitly for this test.